### ISSUE TITLE
New File from Template vertical scrollbar overlaps horizontal lines

### ISSUE THREAD

**Issue by mjross** [Komodo/KomodoEdit#3171]
### Summary
In the File > New > File from Template dialog, the vertical scrollbar area and elevator are on top of the horizontal line separating the list of template types the user can choose from for a new file. In other words, the lines pass underneath the scrollbar elevator, as if the scrollbar were never given an area and background color of its own.

### Steps to Reproduce
Help > Troubleshooting > Restart in Safe-Mode. File > New > File from Template.

### Platform Information
Komodo IDE 11.0.0 build 90668 on Windows 10 Pro

### Attachments
![new file from template dialog](https://user-images.githubusercontent.com/829492/30941774-664b81f0-a3a4-11e7-8943-77e989128419.jpg)

**Comment by Naatan** [comment]
This is by design. It looks far more awkward if it had its own bar reserved unfortunately.

**Comment by mjross** [comment]
If the vertical bar had its own background color, as it does in all of the other Komodo IDE dialogs, then I can't imagine it would look this awkward. Then again, I'm not a web designer.

**Comment by Naatan** [comment]
> as it does in all of the other Komodo IDE dialogs

It doesn't. This is a design convention used throughout Komodo's UI. All scrollbars are floating.

**Comment by mjross** [comment]
What I am saying is that this is the only instance I know of where the background of the window extends through the vertical scrollbar (if that's what you mean by "floating", as in floating above). For instance, in the attached screenshot showing the Preferences window, in the left column, you can see that "Syntax & Spell Checker" does not extend into the vertical scrollbar area. If memory serves, that's what I have seen throughout the UI. Another example is the editor window, where the text of the file does not extend into the vertical scrollbar area, but wraps before doing so – in this example, the "MB;" is wrapped onto a second line instead of continuing into the scrollbar area. Anyway, it isn't critical. I just wanted to point it out because it's the only place in the UI that I've seen where it looks this strange.

![vertical scrollbars](https://user-images.githubusercontent.com/829492/31024032-ffa04406-a4fa-11e7-8011-7fa131f23228.jpg)

![editor window](https://user-images.githubusercontent.com/829492/31024356-037c2760-a4fc-11e7-8c43-5d68d2ae2180.jpg)

**Comment by Naatan** [comment]
That's not exactly true, it also does this for your side pane widgets. Again though I feel like this is a design limitation we're not going to get around. I could have it render a dedicated scrollbar at all times but then we would be back to the design issue the floating scrollbar solved: scrollbars EVERYWHERE. Especially with the side panes this really added up to a ton of wasted screen space and ugly scrollbars on screen whenever you were coding. I think the floating scrollbars strike a nice balance.

**Comment by mjross** [comment]
You're right about the widgets in the side pane. I hadn't noticed that before, probably because I had never before expanded any of those sections enough to force content below the dialog.

![widgets](https://user-images.githubusercontent.com/829492/31025349-99587646-a4ff-11e7-8452-ef70b889f9be.jpg)

But I'm definitely not advocating vertical scrollbars being present even when not needed, i.e., in situations where there is no content that goes below the bottom of the dialog. I'm just talking about those cases where there truly is a need for a vertical scrollbar and one is present.

**Comment by Naatan** [comment]
I see. I can definitely look into whether that is feasible. Unfortunately the scollbar code is all very restrictive, so it may simply not be doable. Will find out.
